rust/phoenix-sdk/src/sdk_client.rs

```diff
@@ -280,23 +280,31 @@ impl SDKClient {
             .await
             .unwrap_or_default();
         let default = Orderbook::<FIFOOrderId, PhoenixOrder> {
-            size_mult: 0.0,
-            price_mult: 0.0,
+            raw_base_units_per_base_lot: 0.0,
+            quote_units_per_raw_base_unit_per_tick: 0.0,
             bids: BTreeMap::new(),
             asks: BTreeMap::new(),
         };
         if market_account_data.is_empty() {
             return Ok(default);
         }
         let (header_bytes, bytes) = market_account_data.split_at(size_of::<MarketHeader>());
-        let base_lots_multiplier = self.base_lots_to_base_units_multiplier(market_key)?;
-        let ticks_multiplier = self.ticks_to_float_price_multiplier(market_key)?;
+        let meta = self.get_market_metadata_from_cache(market_key)?;
+        let raw_base_units_per_base_lot =
```

Comment:
why isn't this number instead `base_lots_per_raw_base_unit`? or is that also not guaranteed to be an integer?

Reply:
This is the number that's multiplied to base lots to return a human readable price

Reply:
In the majority of cases it's going to be < 1